Add page for March-May 2025 presentations (#53)
g-adopt/g-adopt.github.io: docs/events/workshops/posts/2025-visits.md

---
date: 2025-03-28
---

# G-ADOPT Presentations — March–May 2025

![2025 Sydney](../../../images/events/2025_sydney.webp "2025 Sydney"){: style="height:108px" loading=lazy }

**Sydney, Earthbyte group — 28 March 2025**
Rhodri Davies and Sia Ghelichkhan visited the Earthbyte group at the University of
Sydney. They presented recent developments targetting global geodynamical models and
their interaction with plate-reconstruction frameworks, here via GPlates and its
PyGPlates Python interface. A particular focus of this workshop was model output, with
an emphasis on the tools available to post-process and interrogate global 3-D spherical
models.

**Paris, IPGP — 09 May 2025**
Thomas Duvernay visited IPGP and gave an overview of Firedrake and G-ADOPT to institute
members. The presentation included an introduction with hands-on examples of
Firedrake-based simulations alongside a discussion and overview of recent G-ADOPT
developments. The objective was to provide an as-complete-as-possible first look of
Firedrake and G-ADOPT to potential new users.

**Zurich, ETH — 09 May 2025**

> Sia Ghelichkhan visited ETH Zurich.

Reviewer: ```suggestion 
**Munich, LMU — 06 May 2025**
Sia Ghelichkhan visited the Computational Geodynamics group at LMU Munich and presented recent progress on the G-ADOPT platform. The visit included discussions on potential collaborations across multiple fronts, including adjoint methods and links to mineral physics models, with an emphasis on expanding the platform’s applicability to broader geodynamic research.

**Zurich, ETH — 10 May 2025**
Sia Ghelichkhan visited the Seismology and Wave Physics group at ETH Zurich~ to present the initial results from the GeoDynamic Preliminary Reference Earth Model, developed using the REVEAL workflow. The visit provided an opportunity to gather feedback and discuss future integration of seismic and geodynamic data in joint modeling efforts.
```

---
date: 2025-03-28
---

> # G-ADOPT Presentations — March–May 2025

Reviewer 2: ```suggestion
# G-ADOPT Presentations and Tutorials — March–May 2025
```

![2025 Sydney](../../../images/events/2025_sydney.webp "2025 Sydney"){: style="height:108px" loading=lazy }

**Sydney, Earthbyte group — 28 March 2025**
Rhodri Davies and Sia Ghelichkhan visited the Earthbyte group at the University of
Sydney. They presented recent developments targetting global geodynamical models and

> their interaction with plate-reconstruction frameworks, here via GPlates and its

Reviewer 2: Drop "here"

PyGPlates Python interface. A particular focus of this workshop was model output, with
an emphasis on the tools available to post-process and interrogate global 3-D spherical
models.

**Munich, LMU — 06 May 2025**

> Sia Ghelichkhan visited the Computational Geodynamics group at LMU Munich and presented

Reviewer 2: ```suggestion
Sia Ghelichkhan visited the Computational Geodynamics group at LMU Munich and provided an overview of
```

recent progress on the G-ADOPT platform. The visit included discussions on potential
collaborations across multiple fronts, including adjoint methods and links to mineral
physics models, with an emphasis on expanding the platform's applicability to broader
geodynamic research.

**Paris, IPGP — 09 May 2025**
Thomas Duvernay visited IPGP and gave an overview of Firedrake and G-ADOPT to institute
members. The presentation included an introduction with hands-on examples of
Firedrake-based simulations alongside a discussion and overview of recent G-ADOPT
developments. The objective was to provide an as-complete-as-possible first look of
Firedrake and G-ADOPT to potential new users.

**Zurich, ETH — 10 May 2025**
Sia Ghelichkhan visited the Seismology and Wave Physics group at ETH Zurich to present

> the initial results from the GeoDynamic Preliminary Reference Earth Model, developed

Reviewer 2: ```suggestion
the initial results from the GeoDynamic Preliminary Reference Earth Model (G-PREM), developed
```
